WordPress/Classes/Services/MenusService.h

```diff
@@ -0,0 +1,103 @@
+#import "LocalCoreDataService.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class Blog;
+@class Menu;
+@class MenuLocation;
+@class MenuItem;
+
+typedef void(^MenusServiceSuccessBlock)();
+typedef void(^MenusServiceCreateMenuRequestSuccessBlock)(NSNumber *menuID);
+typedef void(^MenusServiceUpdateMenuRequestSuccessBlock)();
+typedef void(^MenusServiceMenusRequestSuccessBlock)(NSArray<Menu *> * _Nullable menus);
```

Comment:
Just curious... can we use `nullable` before the type name like in the regular methods?